doc/classes/@GlobalScope.xml

```diff
@@ -1289,7 +1289,8 @@
 			<return type="Variant" />
 			<param index="0" name="string" type="String" />
 			<description>
-				Converts a formatted [param string] that was returned by [method var_to_str] to the original [Variant].
+				Converts a formatted [param string] that was returned by [method var_to_str] to the equivalent [Variant], without decoding objects.
+				[b]Note:[/b] If you need object deserialization, see [method str_to_var_with_objects].
```

Comment:
Is "(de)serialization" the correct term for a text representation? Someone said that serialization only applies to the binary representation.

Reply:
https://en.wikipedia.org/wiki/Serialization
Wikipedia says it just means a format that can be sent over a network or stored to a file, and converted back. Also mentions XML and other text formats further down the article. So yes, it's correct.